system/jlib/jlog.cpp

```diff
@@ -2453,14 +2446,21 @@ void setupContainerizedLogMsgHandler()
             theManager->changeMonitorFilter(theStderrHandler, filter);
         }
 
-        bool useLogQueue = logConfig->getPropBool(useLogQueueAtt, useQueueDefault);
-        if (useLogQueue)
+        unsigned queueLen = logConfig->getPropInt(logQueueLenAtt, queueLenDefault);
+        if (queueLen)
         {
-            unsigned queueLen = logConfig->getPropInt(logQueueLenAtt, queueLenDefault);
-            unsigned queueDrop = logConfig->getPropInt(logQueueDropAtt, queueDropDefault);
-
             queryLogMsgManager()->enterQueueingMode();
-            queryLogMsgManager()->setQueueDroppingLimit(queueLen, queueDrop);
+            unsigned queueDrop = logConfig->getPropInt(logQueueDropAtt, queueDropDefault);
+            if (queueDrop)
+            {
+                queryLogMsgManager()->setQueueDroppingLimit(queueLen, queueDrop);
+                PROGLOG("JLog: queuing enabled with dropping: queueLen=%u, queueDrop=%u", queueLen, queueDrop);
```

Comment:
somewhat subjective and likely to change, but this is how PROLOG and INFOLOG have been defined:
    MSGCLS_information = 0x08, // Config, environmental and internal status  info
    MSGCLS_progress    = 0x10, // Progress of workunits. Status of file operations
https://github.com/hpcc-systems/HPCC-Platform/blob/a1240fda33e1110602607e5ab876ab4700b17238/system/jlib/jlog.hpp#L66

Reply:
Fair enough, however, PROGLOG is used widely for this purpose, I'd say it is used more for 'internal status'/progress than it is for "Progress of workunits. Status of file operations", but as you say, all a bit subjective.

In fact, it looks like they're being used the opposite way around from what the comments pescribe.
MCuserInfo is being used in 22 places at the moment, and they're mostly related to workunit/files.
PROGLOG is used in 1,624 places, mostly for internal status info purposes.

Perhaps we should flip the comments in jlog.hpp at this stage? (not this PR)